judging/scripts/generate_predictions.sh

```diff
@@ -25,16 +26,26 @@ predictions_file="$repo_dir/tasks/tasks.csv"
 generate_predictions_wrapper="$repo_dir/judging/generate_predictions_local.py"
 prediction_module="$HOME/work/predict.py"
 
+# Get latest validation module from base library (sandboxes in general have out of date base libraries)
+# Overlays the current predictor_validation.py module in the local repo clone dir
+module_path="covid_xprize/validation/predictor_validation.py"
+validation_module="$HOME/work/covid-xprize/$module_path"
+branch="validation-main"
+wget --quiet \
+  --output-document "$validation_module" \
+  https://raw.githubusercontent.com/leaf-ai/covid-xprize/$branch/$module_path
+
 # Run script
 # Need to set up these env vars as cron has a restricted PATH by default
 export PATH=/usr/local/bin/:$PATH
-export PYTHONPATH=/usr/local/lib/python3.7/site-packages
-which python
-which pip
+export PYTHONPATH=/usr/local/lib/python3.7/site-packages${PYTHONPATH:+:$PYTHONPATH}
+command -v python
+command -v pip
 python --version
 pip --version
-# TODO: This next line is probably not needed for the sandbox since pandas is already present
-pip install pandas==1.1.4
+# Make sure minimum version of Pandas is present
+pip install pandas\>=1.1.4
```

Comment:
Controversial? This allows us to _not_ downgrade from a higher version of Pandas that teams might have installed.